docs/content/tutorial/the_end.ngdoc

```diff
@@ -9,11 +9,18 @@ previous steps using the `git checkout` command.
 For more details and examples of the Angular concepts we touched on in this tutorial, see the
 {@link guide/ Developer Guide}.
 
-When you are ready to start developing a project using Angular, we recommend that you bootstrap
-your development with the [angular-seed](https://github.com/angular/angular-seed) project.
+When you are ready to start developing a project using AngularJS, we recommend that you bootstrap
+your development with the [angular-seed project][angular-seed].
 
-We hope this tutorial was useful to you and that you learned enough about Angular to make you want
-to learn more. We especially hope you are inspired to go out and develop Angular web apps of your
-own, and that you might be interested in {@link misc/contribute contributing} to Angular.
+We hope this tutorial was useful to you and that you learned enough about AngularJS to make you want
```

Comment:
I used `AngularJS` when I felt that the framework as a whole was referred to or where a distinction with Angular2 (or AngularDart) should be made and used just `Angular` in all other places. (FWIW, I have thought about every occurence and made a conscious choice for `Angular` vs `AngularJS` - "conscious" doesn't necessarily mean "most appropriate" :stuck_out_tongue: )

If you think it's better to use `Angular` thoughout, I'm fine as well.